src/hb-ot-cff1-table.cc

```diff
@@ -158,6 +245,18 @@ hb_codepoint_t OT::cff1::lookup_expert_subset_charset_for_sid (hb_codepoint_t gl
     return 0;
 }
 
+hb_codepoint_t OT::cff1::lookup_expert_charset_for_glyph (hb_codepoint_t sid)
+{
+  const auto  *pair = hb_bsearch (sid, expert_charset_sid_to_gid, ARRAY_LENGTH (expert_charset_sid_to_gid));
```

Comment:
You can also try `hb_array (expert_charset_sid_to_gid).bsearch (sid)`.  Same below.

Reply:
ah good to know

Reply:
#2349